compiler.go

```diff
@@ -640,6 +640,42 @@ func (c *Compiler) SetImportDir(dir string) {
 	c.importDir = dir
 }
 
+// SetImportFileExt sets the extension name of the source file for loading
+// local module files.
+//
+// Use this method if you want other source file extension than ".tengo".
+// Note that this will replace the current list of extension name.
+//
+//     // this will search for *.tengo, *.foo, *.bar
+//     err := c.SetImportFileExt(".tengo", ".foo", ".bar")
+//
+func (c *Compiler) SetImportFileExt(exts ...string) error {
+	if len(c.importFileExt) == 0 {
+		// At least one extension name is required.
+		c.importFileExt = []string{SourceFileExtDefault}
+	}
+
+	if len(exts) == 0 {
+		return nil // do nothing
+	}
```

Comment:
why are we resetting `c.importFileExt` first? I think this function should just fail and return an error if `len(ext) == 0` (without doing anything).

Reply:
> why are we resetting `c.importFileExt` first? 

You got the point.

I thought, in the first place, was for backward compatibility. In case the `Compiler` object wasn't created via `tengo.NewCompiler()` constructor. Now I realize this is not the right place to initialize, and it should not do more than the function name explains. 

> I think this function should just fail and return an error if `len(ext) == 0` (without doing anything).

That means that the function must contain at least one arg on use. If that's so, shouldn't the syntax be like:

```diff
- func (c *Compiler) SetImportFileExt(exts ...string) error {
+ func (c *Compiler) SetImportFileExt(ext string , exts ...string) error {
```

So to let the user know at least one arg is a must. Or just documenting so will do enough? What do you think?

Reply:
Committed ad168e1 and it seems enough explanation, doesn't it?